Spaceport Culture (#5437)
endless-sky/endless-sky: data/hai/culture hai.txt

# Copyright (c) 2015 by Michael Zahniser
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

# "Missions" which add culture events to the spaceport.

> 

Reviewer: Disclaimer: I absolutely love most of this, but I'm still going to be ruthless in the criticism so it can be made even better.

############################################
#          No Translator Required          #
############################################

mission "ZZCU Theater Chat"
	minor
	source
		government "Hai"
		not attributes "station"
	destination
		distance 1 3
		government "Hai"
	to offer
		random == 1
	on offer
		conversation
			`Walking through the spaceport you come across the uncommon pairing of a Hai and a human, their conversation standing out by virtue of speaking the common tongue of humanity.`

> 			`	"Oh, oh, Gemanari this looks interesting!" The human woman looks to be in her late sixties and, when you turn to look, seems to be clucking over a holographic advertisement for a theater show. "This looks adorable doesn't it? Oh and it says they're using ripple ray refractive holographic flight effects!"`

Reviewer: Missing comma after "Gemanari" and "adorable" and "Oh"

			`	The advertisement does indeed look very impressive and you pause to observe for a moment.`
			`	The Hai, presumably Gemanari, also appears to be a more mature lady who squints at it and wrinkles her nose. "Ugh, that's the Germontian interpretation though."`

> 			`	This receives a puzzled look to which Gemanari thinks for a second before clarifying, "You would say the Darwinist interpretation."`

Reviewer: Make her say social Darwinist

> 			`	This receives a puzzled look to which Gemanari thinks for a second before clarifying, "You would say the Darwinist interpretation."`

Author: Alternate change made.

			`	The lady grimaces and makes like she's vomiting, "I can't believe you still have those schools of thought in a society this old."`
			`	"Oh we don't really, at least not in scientific circles, but a compelling story will outlive such things. I can't stand it personally but it is a classic." When the lady looks unimpressed Gemanari titters and takes her gently by the arm to steer her away again. "Don't worry Alinta, I think the Misheep Wah'uden company is running my preferred interpretation on <destination> at the moment. We can see if they have any cheap tickets. Of course my aunt would insist that the hundred and eighty-third version is the only real one but she always..."`
			`	You lose the conversation as they trail away from you, leaving you to wonder just what was so special about the hundred and eighty-third version of a play about a prehistoric flying Hai found frozen in the ice.`
				decline

mission "ZZCU Musical Entertainment Expo"
	minor
	source
		government "Hai"
		not attributes "station"
	to offer
		random == 1
	on offer
		conversation

> 			`While stopped in the Spaceport waiting for refueling you are surprised by how hard it is to find somewhere to sit and have a meal & a drink. It seems that Hai are crammed into every bar or similar venue with a public display screen, arrayed in sashes and mantles and all manner of other paraphernalia which, near as you can tell looks like... team sport merchandise?`

Author: I'm going to keep 'stopped', have accepted the rest.

> 			`	You're not really an expert on exactly what such things look like in Hai society and so when you finally find somewhere to sit, at a bar for lack of any other options, you ask the tired looking bartender what exactly is going on.`

Author: The one time I go for simpler language and someone suggests I use more advanced vocabulary. :P 

While I would use -'s personally in messages, I think that's frowned upon in the ES style directions.

			`	"Oh this? They're all here for the Musical Entertainment Expo streams. Unfortunately, none of them start drinking until at least one of their teams has announced something of worth," he says with an exasperated eye-roll.`
			choice
				`	"Sure, can I just get some food here?"`
					goto wrapup
				`	"Whats the Musical Entertainment Expo?"`
				`	"Why don't they start drinking yet?"`

> 			`	"Well you see the big music labels do a lot of talent scouting all year round, they all have a department that is solely dedicated to finding and assembling new music groups to be the next big thing. It's traditionally to celebrate only after you've committed to the journey with a new band. Every three years the industry holds ME2, that's what you're seeing up there, to debut their next round of bands, and to tease any mid-season releases they might have in the works."`

Reviewer: Missing comma after "you see".
Missing conjunction before "they all have".
Why would a Hai event have an acronym that makes sense in English?
Replace "that's what you're seeing" with "which is what you're seeing".

Author: I disagree with the missing conjunction, it's speech after all and it's not necessary to include it when spoken.
It's heavily implied that the Hai have a pretty decent education system and learn Human standard - I don't see it as much of a reach that they could re-acronym a name after they've translated the name across.

> 			`	"Well you see the big music labels do a lot of talent scouting all year round, they all have a department that is solely dedicated to finding and assembling new music groups to be the next big thing. It's traditionally to celebrate only after you've committed to the journey with a new band. Every three years the industry holds ME2, that's what you're seeing up there, to debut their next round of bands, and to tease any mid-season releases they might have in the works."`

Author: Cheers, and the choice of ME2 is a subtle reference to E3.

			choice
				`	"Your music comes off a production line?"`
					goto production
				`	"Do you not have musicians trying to make it on their own?"` 
			`	"Oh for sure, but mostly just to get picked up by one of these companies."`
			label production

> 			`	The bartender chuckles. "We've been doing pop music for longer than your people have had fire, it's about the experience of being a fan of a group as it rises to fame and stardom. A dedicated Hai can go back and find a song for almost any possible life experience in almost any possible genre. It's the journey, and the personalities, that are what it's all about."`

Author: In this particular case I'm going to disagree, but only because of the intended emphasis.
The entire paragraph is in the context of 'the experience'. 
`being a fan as a group rises` shares the emphasis with the group, the original links the event of `it rising` to `the experience`.
Similarly the language choice in the last sentence is meant to emphasise that it is specifically these things that are are the purpose, and detract from other factors.
I could change 'that are what it's all about' though, I sorta get that that might be excessive.

			choice
				`	"How strange."`
				`	"How interesting."`
			`	"Anyway, I gather you were you after some food?"`
			label wrapup
			`	The bartender tips his head in your direction and takes your order. "No problem, it'll be right up."`
			`	It's a noisy meal.`
				decline

#This uses the flag from receiving the Hai translator.
mission "ZZCU The Meme Wars 1"
	minor
	source
		government "Hai"
		not attributes "station"
	to offer
		random == 1
	on offer
		conversation

> 			`Stepping into the spaceport you are beset by a riot of color. Every blank surface that's not usually paid advertising space seems to be festooned with dangling holographic projectors, ribbons, banners, and even actual paper fliers and posters, apparently taped to whatever surface will bear them. At first you wonder if there's some sort of festival going on, but as you watch a young Hai wearing a hood slaps up another holo projector which immediately projects the image of an anthro root vegetable with a sword hacking at some kind of impressionist concept of a Hai elder made up of iridescent triangles. The young Hai then proceeds to tear down a banner and slap some goo on a massive poster which begins dissolving before a pair of security guards, wearing some very obviously non-standard additions to their uniforms, chase him back out of the spaceport.`

Reviewer: Use "anthropomorphic". If you use "anthro" the first thing the player's going to think is "Is the dev a furry?".
The banner-tearing and goo-slapping are a bit too fast-paced and out of the blue to look elegantly written.

Author: Changed anthro, and it's meant to be fast-paced and inelegant. The idea is that it's happening almost too quickly for you to follow exactly what's going on because you've just walked into the equivalent of a warzone at an anime gaming convention.

> 			`Stepping into the spaceport you are beset by a riot of color. Every blank surface that's not usually paid advertising space seems to be festooned with dangling holographic projectors, ribbons, banners, and even actual paper fliers and posters, apparently taped to whatever surface will bear them. At first you wonder if there's some sort of festival going on, but as you watch a young Hai wearing a hood slaps up another holo projector which immediately projects the image of an anthro root vegetable with a sword hacking at some kind of impressionist concept of a Hai elder made up of iridescent triangles. The young Hai then proceeds to tear down a banner and slap some goo on a massive poster which begins dissolving before a pair of security guards, wearing some very obviously non-standard additions to their uniforms, chase him back out of the spaceport.`

Author: I have altered the first paragraph for clarity incorporating some of your suggestion.
The last bit I'm hesitant to change because 'continues to vandalize' sounds like the Hai has time to do several things, when I want to give the impression that they had only literal seconds before security started chasing them.

			branch translator
				has "language: Wanderer"

> 			`	As you watch the triangle elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance, before proceeding to act out something that makes precisely no sense to you.`

Reviewer: Rewrite this from the ground up, it makes no sense.

Author: I assume you're referring to line 86, it's an e-warfare device inside a holoprojector. When the projected memes do battle the underlying computers are also trying to kill each other. One wins.

Reviewer: I mean that it's written and conveyed poorly.

Author: It has edited based on vkr's feedback, so maybe look there, but beyond that I'm afraid unless you can be more specific I can't see particularly what the complaint is.

> 			`	As you watch the triangle elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance, before proceeding to act out something that makes precisely no sense to you.`

Author: It's not coincidental, there's e-warfare happening in the background.
`triangular` is definitely a good catch, will make a pass on that.

			`	A little further on you encounter two different groups of protesters weaving in and out of the spaceport's entrances like some kind of conga line. The triangular elder and root vegetable make several reappearances on their placards, some of them holographically animated.`

> 			`	A different security guard, this one in regular attire, appears at your elbow. "You look a little lost sir but don't worry, it's just a meme war. Carry on with your business and don't touch the meme items, no one should bother you over it."`

Reviewer: I hate to say "cringe" in 2021, but **this is cringe.**
Aside from the usual wonky pacing, DO NOT USE THE WORD "MEME" IN THE GAME PLEASE I BEG YOU.

*cough* sorry i lost my cool

Author: Meme might be popularised right now, but it's a legitimate term to describe the phenomenon. In fact there is no equivalent term in english which can be substituted and this is a rare case where society has popularised the correct scientific use of a term.

			`	You thank the security guard for his advice and go about what you were doing as the baffling conflict continues around you.`
				decline
			label translator

> 			`	As you watch the triangle elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance. To your surprise your translator pipes up, "[Victory, Success] for the true and square root that is more [significant, meaningful] than the tripartite [system, tradition] of the elders!"`

Reviewer: Translator output is weird, but it could be excused by it not being very good at picking up Hai slang.
Do the Hai even have slang?

Author: This is how the translator translates Hai, you only otherwise use the translator when the Wanderer's speak Hai to you, because the Hai you encounter generally have learned human standard.

Reviewer: I mean that "for the true and square root that is more [significant, meaningful]" is an awkward construction.

Author: Ah yes, well in this case consider your translator is essentially transliterating meme text.

			choice
				`	"How strange."`
				`	"Sounds fun."`
			label done
			`	You thank the security guard for his advice and go about what you were doing as the baffling conflict continues around you.`
				decline

mission "ZZCU Wrestling Star"
	minor
	source
		government "Hai"
		not attributes "station"
	to offer
		random == 1
	on offer
		conversation
			`A famous Hai wrestler is doing a signing. (TBD)`
				decline

mission "ZZCU Fencing Star"
	minor
	source
		government "Hai"
		not attributes "station"
	to offer
		random == 1
	on offer
		conversation
			`A famous Hai fencer is doing a signing. (TBD)`
				decline
				
mission "ZZCU ??? Star"
	minor
	source
		government "Hai"
		not attributes "station"
	to offer
		random == 1
	on offer
		conversation
			`A famous Hai sportsperson of an obscure alien sport is doing a signing. (TBD)`
				decline

mission "ZZCU Gambling Judgment"
	minor
	source
		government "Hai"
		not attributes "station"
	to offer
		random == 1
	on offer
		conversation

> 			`Upon entering a diner at the spaceport you have not been seated long before you notice there is a group of other human ship captains sequestered in a corner playing, near as you can tell, a game of poker. You note that the group seems to be getting a variety of dark looks from various Hai also dining at the establishment.`

Reviewer: "Upon entering" and "you have not been seated long" directly conflict.
I was not aware "near as you can tell" was a thing, use "as far" instead possibly.

Author: Re-done the first part.
R.E. "as far" -> I have a little bit of a personal crusade against simplifying language for the sake of simplifying language in a _text based_ game. So if the phrasing is legitimate and the meaning is clear, I'm in favour of keeping it.

> 			`Upon entering a diner at the spaceport you have not been seated long before you notice there is a group of other human ship captains sequestered in a corner playing, near as you can tell, a game of poker. You note that the group seems to be getting a variety of dark looks from various Hai also dining at the establishment.`

Author: I think the original phrasing emphasises that the group of human captains are a minority in the room. I'm not against this change though if people want it.

			`	After a short while the barkeep, a middling and slightly overweight Hai, comes over to take your order.`
			choice

> 				`	"Just some food thanks." (Place an order.)`

Reviewer: Aside from the missing comma, that's a weird order. I suggest just having (Place an order.)

					goto end

> 				`	"Hey, I notice there are some dark looks around here. Are human captains not welcome?"`

Reviewer: Repetition of "dark looks", wrong tense on "I notice".

Author: If you noticed what you'd think of as 'dark looks' would you not then use the words 'dark looks' when asking about it?
Fixed the tense.

Reviewer: I don't care if you would realistically use the word, repetitions are bad.
The player will see the repetition and see it as ugly.

> 			`	The barkeep glances over at the table and shakes his head, "Oh no, you're perfectly welcome. That group is just breaking the maturity tenets, some Hai get uncomfortable with adults blatantly acting like children in public."`

Author: I have adopted the second part of this suggestion, I think the experience of the Hai is sufficient to say that <first> would have to be a dumbass to assume it might be 'all humans'.

			choice

> 				`	"OK..." (Place an order.)`

Reviewer: No idea why but "OK" seems out of place for a conversation option in ES. Just a gut feeling.

Author: Removed for consistency with previous choice.

					goto end
				`	"The maturity tenets?"`

> 			`	"Oh right, I suppose you wouldn't know, there are a number of tenets that Hai are taught growing up, lessons to be internalized to be a responsible and functioning member of society. The second is 'To be honest in dealings.' and the third is 'To gamble only on skill, not chance.' so their game breaks two of them."`

Reviewer: Replace comma with period after "I suppose you wouldn't know".
"to be internalized to be" has a repetition but it isn't particularly bad.
Put commas after quotes: "The second is 'To be honest in dealings.', and the third is 'To gamble only on skill, not chance.', so their game breaks two of them."

Author: I knew I'd bugger the quotes somehow.

			choice
				`	"Some would say Poker is more about skill than chance."`

> 				`	"Surely if it's just a game though, there's no harm in it?"`

Reviewer: It's fine but maybe could sound better if it was more like:
"Surely though, if it's just a game, there's no harm in it?"

Author: I personally, when speaking aloud, would only use the two-part sentence.
I agree though, that if it was written as if it were writing, the three-part structure makes sense.

> 			`	"Oh no, I'm afraid the first tenet ensures that it's really not favorable either way. The first is 'To have clear separation of fantasy and reality.' so a game cannot have real consequences or it is reality, not fantasy. So to gamble money on the game is to bring real consequences to a fantasy exercise, this is taboo for an adult.`

Author: I'm just going to say - People don't necessarily speak and present information logically.
But if you think there are a collection of grammar edits needed please use the function to suggest en edit.

> 			`	"Oh no, I'm afraid the first tenet ensures that it's really not favorable either way. The first is 'To have clear separation of fantasy and reality.' so a game cannot have real consequences or it is reality, not fantasy. So to gamble money on the game is to bring real consequences to a fantasy exercise, this is taboo for an adult.`

Reviewer: > I'm just going to say - People don't necessarily speak and present information logically.

This is a terrible approach to writing anything. In movies people don't interrupt others, forget what they're saying, etc. unless it serves the plot. If someone coughs in a story they are either dying or they have some disease which is important to the plot. This is not a flaw, it is by design. You shouldn't excuse wonky writing with "realism", because if you actually wanted to write exactly how people speak it would suck to read.

			`	"If you interpret the game as being played for the skill, without the wagering, then you clear the first tenet, but you have the issue that the principle skills are bluffing and manipulation, which violates the second tenet. Bluffing isn't a skill seen as having virtue, nor is manipulation."`
			choice
				`	"Wait, do the Hai not lie or bluff?"`
			`	The barkeep chortles with a strange trill and says, "Oh no the Hai do lie and bluff, we're actually very good at bluffing, evolutionarily advantageous in the wild and all that. It's just seen as very immature and base. An adult should be able to conduct their lives without resorting to such primitive instincts. We learned that the hard way long ago, back before we holed up in our current borders."`
			`	He looks at you intently before continuing, "You won't hear the tenets brought up very much, but if you look closely you'll see that the first three especially are ingrained in our society very deeply."`

> 			`	With a shrug he flips open his order pad and pats his belly. "Of course, my gran always said that any of them after the tenth were more guidelines than actual rules, and I certainly have my share of disagreements with thirteen. Now, before I talk your ears of, what'll I get for you?"`

Reviewer: Did you mean "the thirteen*th*"?
Misspelling of "off".

Author: I did not, it is common linguistic shorthand to refer to familiar entities on a list of distinct and individually separate items by 'name' rather than by position. 'Thirteen' is the 'name' of the thirteenth tenet.
Fixed spelling.

Reviewer: Use "the thirteen" or "number thirteen". It's just ugly how it is right now.

> 			`	With a shrug he flips open his order pad and pats his belly. "Of course, my gran always said that any of them after the tenth were more guidelines than actual rules, and I certainly have my share of disagreements with thirteen. Now, before I talk your ears of, what'll I get for you?"`

Author: See response to trerri re this.

			label end

> 			`	He takes your order diligently and with a smile.`

Reviewer: I don't seem to recall that the ending label ever works like this, it's usually just (done) or something in the first person.

Author: The label endings are just code hooks in the background. They're just matching strings for the goto-label combo.

Reviewer: No no, I was referring to "He takes your order diligently and with a smile."
No other conversation ends in the second person like that.

Author: Oh, hmmm.

Author: Ah right, so you can't go directly from a label to a `decline` to end. So in order for you to skip the explanation the mission _needs_ some text there.
If you can think of an alternative way of doing it please submit a suggestion. :)

Reviewer: Just have the final one also say (Place an order.) and when you choose it it just says 'The bartender takes your order with a smile.'

Author: I don't see how what you're proposing is meant to work. Why don't you _insert a suggestion_ which demonstrates it?

# Copyright (c) 2015 by Michael Zahniser
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

# "Missions" which add culture events to the spaceport.

############################################
#          No Translator Required          #
############################################

> mission "ZZCU Theater Chat"

Reviewer 2: It seems to me like the strict offer conditions of most of these missions fulfil the purpose of making sure they don't get offered over other missions. Even ignoring that though, I don't see an issue with sometimes getting these missions over other missions. It's not like these ones repeat. Therefore, I wouldn't have these extra characters in the mission names just to push the mission to the end of the offer list. 

I'd remove the comments, like the one above. They seem unnecessary to me. There's no need to comment to organize these missions, or to say who worked on them, since we know that from the PR and ultimately the changelog attribution. Whatever the case on comments, we should have ones like the "No Translator Required" comment above, as that actually goes against the coding style guide, so I'd avoid it in the data files as well.

> Fancy, decorated comments should not be used.
> For example, perhaps your editor has a "feature" that lets you automatically insert comments like this:
```
/*************************************
 * My fancy comment.
 * This is a comment that could have
 * been a single line without these
 * fancy decorative elements.
 *************************************/
```
> But, those two lines of asterisks are eye-grabbing and completely unnecessary on any editor that supports syntax coloring, and more importantly anyone using a different editor would have to manually type an asterisk at the beginning of every comment line and figure out exactly how many asterisks are in those rows at the top and bottom, in order to be consistent with your code.

Having separate files just for these missions also seem unnecessary.  These could simply be put into the existing mission files. "culture hai.txt" gets added to "hai mission.txt," the rest just get added to "transport mission.txt" (even though they aren't actually transport missions, but that file has already had missions added that aren't transport related).

Reviewer 3: "transport missions.txt" = "general purpose mission file for non specific missions"

Author: The ZZ was mostly because the Spider had raised some strident objections to them being in a position to block other missions, since these are all missions that are specifically of no consequence.

This was also why they're in their own files. There's going to be more of them, and all the other missions in the game so far have some sort of consequence. Whether it's a reward, or a change in the world, or they unlock other missions or dialogue options. All the other missions _do_ something, and these only provide world-building insight to the player. There's only 23 missions across the set so far, but Lake is doing some more and the plan is to come back with a second batch and so on until (across this scope) there's about 180 of them. 

I really think it's better if we don't put a clutter of no-consequence fluff missions in the same files with the ones that actually do things. If only for the benefit of content creators and their sanity when trying to sort through things.

Reviewer 2: If we want these is separate files, then at the very least combine all the human ones. The distinction between regions/sources into different files is overkill to me. And how about instead of calling them “culture missions,” we call them “spaceport conversations.txt,” with the Hai one being “hai spaceport conversations.txt”?

	minor
	source
		government "Hai"
		not attributes "station"
	destination
		distance 1 3
		government "Hai"
	to offer
		random < 1
	on offer
		conversation
			`As you are walking through the spaceport, you come across the uncommon pairing of a Hai and a human, their conversation standing out by virtue of speaking the common tongue of humanity.`
			`	"Oh, oh, Gemanari, this looks interesting!" The human woman looks to be in her late sixties and, when you turn to look, seems to be clucking over a holographic advertisement for a theater show. "This looks adorable, doesn't it? Oh, and it says they're using ripple ray refractive holographic flight effects!"`
			`	The advertisement does indeed look very impressive and you pause to observe for a moment.`

> 			`	The Hai, presumably Gemanari, also appears to be a more mature lady who squints at it and wrinkles her nose. "Ugh, that's the Germontian interpretation though."`

Author: *brain pain* what?

> 			`	The Hai, presumably Gemanari, also appears to be a more mature lady who squints at it and wrinkles her nose. "Ugh, that's the Germontian interpretation though."`

Reviewer 4: Whoops. That sentence has some strange phrasing, so I thought it was wrong and subsequently made an error trying to fix it. You're good to leave it as-is then.

			`	This receives a puzzled look to which Gemanari thinks for a second before clarifying, "You would say the Social Darwinist interpretation."`

> 			`	The lady grimaces and makes like she's vomiting, "I can't believe you still have those schools of thought in a society this old."`

Author: I think I meant this more like a subtle mime than a look.

> 			`	The lady grimaces and makes like she's vomiting, "I can't believe you still have those schools of thought in a society this old."`

Reviewer 4: Oh okay. First suggestion should be good then

			`	"Oh we don't really, at least not in scientific circles, but a compelling story will outlive such things. I can't stand it personally but it is a classic." When the lady looks unimpressed Gemanari titters and takes her gently by the arm to steer her away again. "Don't worry Alinta, I think the Misheep Wah'uden company is running my preferred interpretation on <destination> at the moment. We can see if they have any cheap tickets. Of course my aunt would insist that the hundred and eighty-third version is the only real one but she always..."`
			`	You lose the conversation as they trail away from you, leaving you to wonder just what was so special about the hundred and eighty-third version of a play about a prehistoric flying Hai found frozen in the ice.`
				decline

mission "ZZCU Musical Entertainment Expo"
	minor
	source
		government "Hai"
		not attributes "station"
	to offer
		random < 1
	on offer
		conversation

> 			`While stopped in the spaceport waiting for refueling you are surprised by how hard it is to find somewhere to sit and have a meal & a drink. It seems Hai have crammed themselves into every bar or similar venue with a public display screen, arrayed in sashes and mantles and all manner of other paraphernalia which, near as you can tell, looks like... team sport merchandise?`

Reviewer 4: ```suggestion
			`While stopped in the spaceport waiting for refueling, you are surprised by how hard it is to find somewhere to sit & have a meal and a drink. It seems Hai have crammed themselves into every bar or similar venue with a public display screen, arrayed in sashes and mantles and all manner of other paraphernalia which, near as you can tell, look like... team sport merchandise?`
```
```suggestion
			`While stopped in the spaceport waiting for refueling you are surprised by how hard it is to find somewhere to sit and have a meal and a drink. It seems Hai have crammed themselves into every bar or similar venue with a public display screen, arrayed in sashes and mantles and all manner of other paraphernalia which, near as you can tell, look like... team sport merchandise?`
```
Don't remember ever seeing `&` in-game. Was it there just to differentiate it from the `and` right before?

Author: It was yes.

> 			`	You're not really an expert on exactly what such things look like in Hai society and so when you finally find somewhere to sit, at a bar for lack of any other options, you ask the tired looking bartender what exactly is going on.`

Reviewer 4: ```suggestion
			`	You're not really an expert on exactly what such things look like in Hai society and so when you finally find somewhere to sit, at a bar for lack of any other options, you ask the tired-looking bartender what exactly is going on.`
```
`tired-looking` forms a single adjective describing `bartender`, so needs a dash. There's also two `exactly`s here. You could switch one of them out for variety if you wanted. Maybe a `precisely`? Or just take one out? Or nothing at all; your choice.

Author: I think I'd take out the first one.

Reviewer 4: ```suggestion
			`	You're not really an expert on what such things look like in Hai society and so when you finally find somewhere to sit, at a bar for lack of any other options, you ask the tired looking bartender what exactly is going on.`
```

			`	"Oh this? They're all here for the Musical Entertainment Expo streams. Unfortunately, none of them start drinking until at least one of their teams has announced something of worth," he says with an exasperated eye-roll.`
			choice

> 				`	"Sure, can I just get some food here?"`

Reviewer 4: The `Sure` feels a little out of place to me because the First Last isn't really affirming anything.
```suggestion
				`	"Okay. Can I just get some food here?"`
```
There's a lot of other alternatives you could do too.

					goto wrapup

> 				`	"Whats the Musical Entertainment Expo?"`

Reviewer 4: ```suggestion
				`	"What's the Musical Entertainment Expo?"`
```

				`	"Why don't they start drinking yet?"`

> 			`	"Well you see, the big music labels do a lot of talent scouting all year round, they all have a department that is solely dedicated to finding and assembling new music groups to be the next big thing. It's tradition to celebrate only after you've committed to the journey with a new band. Every three years the industry holds ME2, which is what you're seeing up there, to debut their next round of bands and to tease any mid-season releases they might have in the works."`

Reviewer 4: ```suggestion
			`	"Well you see, the big music labels do a lot of talent scouting all year round. They all have departments solely dedicated to finding and assembling new music groups to be the next big thing. It's tradition to celebrate only after you've committed to the journey with a new band. Every three years the industry holds ME2, which is what you're seeing up there, to debut their next round of bands and to tease any mid-season releases they might have in the works."`
```
I prefer the period for the longer pause it provides, and I think the other edit is just a little smoother. Again, totally up to you.

			choice
				`	"Your music comes off a production line?"`
					goto production
				`	"Do you not have musicians trying to make it on their own?"` 
			`	"Oh for sure, but mostly just to get picked up by one of these companies."`
			label production

> 			`	The bartender chuckles. "We've been doing pop music for longer than your people have had fire, it's about the experience of being a fan of a group as it rises to fame and stardom. A dedicated Hai can go back and find a song for almost any possible life experience in almost any possible genre. It's the journey, and the personalities, that are what it's all about."`

Reviewer 4: ```suggestion
			`	The bartender chuckles. "We've been doing pop music for longer than your people have had fire. it's about the experience of being a fan of a group as it rises to fame and stardom. A dedicated Hai can go back and find a song for almost any possible life experience in almost any possible genre. It's the journey, and the personalities, that are what it's all about."`
```
Same as above with the pause.

Author: Subsequent capitalisation got missed. :P

Reviewer 4: My bad!
```suggestion
			`	The bartender chuckles. "We've been doing pop music for longer than your people have had fire. It's about the experience of being a fan of a group as it rises to fame and stardom. A dedicated Hai can go back and find a song for almost any possible life experience in almost any possible genre. It's the journey, and the personalities, that are what it's all about."`
```

			choice
				`	"How strange."`
				`	"How interesting."`
			`	"Anyway, I gather you were after some food?"`
			label wrapup

> 			`	The bartender tips his head in your direction and takes your order. "No problem, it'll be right up."`
> 			`	It's a noisy meal.`

Reviewer 4: ```suggestion
			`	The bartender tips his head in your direction and takes your order. "No problem, it'll be right up." It's a noisy meal.`
```
These two paragraphs are especially tiny. Stitch 'em together if ya want.

Author: This separation is for emphasis.

Reviewer 4: Fair enough. Feel free to resolve.

			branch translator
				has "language: Wanderer"
			`	As you watch, the triangular elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance, before proceeding to act out something that makes precisely no sense to you.`
			`	A little further on you encounter two different groups of protesters weaving in and out of the spaceport's entrances like some kind of conga line. The triangular elder and opposing root vegetable make several reappearances on their placards, some of them holographically animated.`

> 			`	A different security guard, this one in regular attire, appears at your elbow. "You look a little lost but don't worry, it's just a meme war. Carry on with your business and don't touch the meme items, no one should bother you over it."`

Reviewer 4: ```suggestion
			`	A different security guard, this one in regular attire, appears at your elbow. "You look a little lost. Don't worry, it's just a meme war. Carry on with your business and don't touch the meme items, and no one should bother you over it."`
```
Optional; thought it helped with flow.

			`	You thank the security guard for his advice and go about what you were doing as the baffling conflict continues around you.`
				decline
			label translator
			`	As you watch, the triangular elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance. To your surprise your translator pipes up, "[Victory, Success] for the true and square root that is more [significant, meaningful] than the tripartite [system, tradition] of the elders!"`
			`	A little further on you encounter two different groups of protesters weaving in and out of the spaceport's entrances like some kind of conga line. The triangular elder and opposing root vegetable make several reappearances on their placards, some of them holographically animated.`

> 			`	A different security guard, this one in regular attire, appears at your elbow. "You look a little lost but don't worry, it's just a meme war. Carry on with your business and don't touch the meme items, no one should bother you over it."`

Reviewer 4: ```suggestion
			`	A different security guard, this one in regular attire, appears at your elbow. "You look a little lost. Don't worry, it's just a meme war. Carry on with your business and don't touch the meme items, and no one should bother you over it."`
```
Copied+pasted from above.

			choice
				`	"Sure, will do."`
					goto done
				`	"Your elders have a tripartite system?"`

> 			`	The security guard chuckles and says, "Oh you can understand? They don't really, but it's a very old joke from an old campaign that there are only three tenets the elders care about, and they're all the ones that affect us the least."`

Reviewer 4: Double old. Maybe `ancient` as an alternative, or remove one? Up to you.

Author: ```suggestion
			`	The security guard chuckles and says, "Oh you can understand? They don't really, but it's a very old joke from a bygone political campaign that there are only three tenets the elders care about, and they're all the ones that affect us the least."`
```

> 			`	When you ask what has triggered this now he says, "Oh, the latest education priorities came out with a four tenet focus, on the same day that an elder accidentally quoted the old campaign. It has been very entertaining."`

Reviewer 4: ```suggestion
			`	When you ask what has triggered this, he says, "Oh, the latest education priorities came out with a four tenet focus, on the same day that an elder accidentally quoted the old campaign. It has been very entertaining."`
```
I'd push for a comma here, but that's just me. The `now` seemed a little confusing. If we're asking what inciting event triggered the meme war, then I don't think it's necessary.

			choice
				`	"How strange."`
				`	"Sounds fun."`
			label done
			`	You thank the security guard for his advice and go about what you were doing as the baffling conflict continues around you.`
				decline

mission "ZZCU Wrestling Star"
	minor
	source
		government "Hai"
		not attributes "station"
	to offer
		random < 1
	on offer
		conversation
			`As you pass through the spaceport you notice a crowd of trilling Hai clustered in one area.`
			choice
				`	(Ignore them and pass by.)`
					goto skip
				`	(Take a look.)`

> 			`	The crowd appears to be mostly made up of younger Hai though a few more mature Hai are also there, visibly exhibiting more restraint on their enthusiasm. Reaching the edge of the crowd and peering over the shorter Hai folk you can see a table set up with two standing banners flanking it and a stack of, at a glance, glossy printed photos.`

Reviewer 4: ```suggestion
			`	The crowd appears to be mostly made up of younger Hai, though a few more mature Hai are also there, visibly exhibiting more restraint on their enthusiasm. Reaching the edge of the crowd and peering over the shorter Hai folk, you can see a table set up with two standing banners flanking it and a stack of, at a glance, glossy printed photos.`
```

			`	The banners depict a rather buff looking Hai, rendered in great detail, with a holographically enhanced animation that makes it appear to be continuously going through the motion of lifting and dropping a generically blurred opponent. The effect is, though, achieved solely through a trick of artificially shifting perspective. To your surprise you realize that this is a signing event for a wrestling star, of all things.`
			`	After only a moment the individual in question emerges from a booth behind the setup and greets their fans, who prove very enthusiastic, before sitting down with some kind of pen for the assembled Hai to line up. It would seem that certain human things have become popularized among the Hai.`
			label skip
			`	You can hear the Hai crowd trilling for some distance as you go about your business.`
				decline

mission "ZZCU Fencing Star"
	minor
	source
		government "Hai"
		not attributes "station"
	to offer
		random < 1
	on offer
		conversation
			`As you pass through the spaceport you notice a crowd of trilling Hai clustered in one area.`
			choice
				`	(Ignore them and pass by.)`
					goto skip
				`	(Take a look.)`

> 			`	The crowd appears to be mostly made up of younger Hai though a few more mature Hai are also there, visibly exhibiting more restraint on their enthusiasm. Reaching the edge of the crowd and peering over the shorter Hai folk you can see a table set up with two standing banners flanking it and a stack of, at a glance, glossy printed photos.`

Reviewer 4: ```suggestion
			`	The crowd appears to be mostly made up of younger Hai, though a few more mature Hai are also there, visibly exhibiting more restraint on their enthusiasm. Reaching the edge of the crowd and peering over the shorter Hai folk, you can see a table set up with two standing banners flanking it and a stack of, at a glance, glossy printed photos.`
```

> 			`	The banners depict a svelte Hai in the strikingly familiar costume of a fencer, with a holographically enhanced animation that makes it appear to be continuously performing an elegant parry and riposte against an out-of-frame opponent. The effect is, though, achieved solely through a trick of artificially shifting perspective. To your surprise you realize that this is a signing event for a fencing star, of all things.`

Reviewer 4: Ooo, `svelte`. Had to look that one up. Nice word choice there

			`	After only a moment the individual in question emerges from a booth behind the setup and greets their fans, who prove very enthusiastic, before sitting down with some kind of pen for the assembled Hai to line up. It would seem that certain human things have become popularized among the Hai.`
			label skip
			`	You can hear the Hai crowd trilling for some distance as you go about your business.`
				decline
				
mission "ZZCU Mystery Sport Star"
	minor
	source
		government "Hai"
		not attributes "station"
	to offer
		random < 1
	on offer
		conversation
			`As you pass through the spaceport you notice a crowd of trilling Hai clustered in one area.`
			choice
				`	(Ignore them and pass by.)`
					goto skip
				`	(Take a look.)`

> 			`	The crowd appears to be mostly made up of younger Hai though a few more mature Hai are also there, visibly exhibiting more restraint on their enthusiasm. Reaching the edge of the crowd and peering over the shorter Hai folk you can see a table set up with two standing banners flanking it and a stack of, at a glance, glossy printed photos.`

Reviewer 4: ```suggestion
			`	The crowd appears to be mostly made up of younger Hai, though a few more mature Hai are also there, visibly exhibiting more restraint on their enthusiasm. Reaching the edge of the crowd and peering over the shorter Hai folk, you can see a table set up with two standing banners flanking it and a stack of, at a glance, glossy printed photos.`
```

> 			`	The banners depict a pair of Hai, one with a net and one with what looks like a bladed frisbee. It's not at all clear what precisely it is that they do and when the individuals in question emerge from a booth behind the setup and greet their fans it becomes clear that, if you could even make yourself heard over the trilling, no one is particularly keen to explain it to you. The pair remove their helmets and flash dazzling Hai smiles at their fans before sitting down with some kind of pen for the assembled Hai to line up.`

Reviewer 4: ```suggestion
			`	The banners depict a pair of Hai, one with a net and one with what looks like a bladed frisbee. It's not at all clear what precisely it is that they do, and when the individuals in question emerge from a booth behind the setup and greet their fans it becomes clear that, if you could even make yourself heard over the trilling, no one is particularly keen to explain it to you. The pair remove their helmets and flash dazzling Hai smiles at their fans before sitting down with some kind of pen for the assembled Hai to line up.`
```

			`	It would seem that celebrity signings are popular among the Hai.`
			label skip
			`	You can hear the Hai crowd trilling for some distance as you go about your business.`
				decline

mission "ZZCU Gambling Judgment"
	minor
	source
		government "Hai"
		not attributes "station"
	to offer
		random < 1
	on offer
		conversation
			`You stop at a diner for a meal grounds-side at the spaceport, but you have not been seated long before you notice there is a group of other human ship captains sequestered in a corner playing, near as you can tell, a game of poker. You note that the group seems to be getting a variety of dark looks from various Hai also dining at the establishment.`
			`	After a short while the barkeep, a middling and slightly overweight Hai, comes over to take your order.`
			choice
				`	(Place an order.)`
					goto end
				`	"Hey, I noticed there are some dark looks around here. Are human captains not welcome?"`

> 			`	The barkeep looks confused for a second before looking over at the table and shaking his head in understanding, "Oh no, you're perfectly welcome. That group is just breaking the maturity tenets, some Hai get uncomfortable with adults blatantly acting like children in public."`

Reviewer 4: ```suggestion
			`	The barkeep looks confused for a second before looking over at the table and shaking his head in understanding. "Oh no, you're perfectly welcome. That group is just breaking the maturity tenets, some Hai get uncomfortable with adults blatantly acting like children in public."`
```

			choice
				`	(Place an order.)`
					goto end
				`	"The maturity tenets?"`
			`	"Oh right, I suppose you wouldn't know. There are a number of tenets that Hai are taught growing up, lessons to be internalized to be a responsible and functioning member of society. The second is 'To be honest in dealings,' and the third is 'To gamble only on skill, not chance,' so their game breaks two of them."`
			choice

> 				`	"Some would say Poker is more about skill than chance."`

Reviewer 4: ```suggestion
				`	"Some would say poker is more about skill than chance."`
```

				`	"Surely if it's just a game though, there's no harm in it?"`
			`	"Oh no, I'm afraid the first tenet ensures that it's really not favorable either way. The first is 'To have clear separation of fantasy and reality,' so a game cannot have real consequences or it is reality, not fantasy. So to gamble money on the game is to bring real consequences to a fantasy exercise. This is taboo for an adult.`

> 			`	"If you interpret the game as being played for the skill, without the wagering, then you clear the first tenet, but you have the issue that the principle skills are bluffing and manipulation, which violates the second tenet. Bluffing isn't a skill seen as having virtue, nor is manipulation."`

Reviewer 4: ```suggestion
			`	"If you interpret the game as being played for the skill, without the wagering, then you clear the first and third tenet, but you have the issue that the principle skills are bluffing and manipulation, which violates the second tenet. Bluffing isn't a skill seen as having virtue, nor is manipulation."`
```
Skill passes 3rd, no wagering passes 1st, right?

			choice
				`	"Wait, do the Hai not lie or bluff?"`

> 			`	The barkeep chortles with a strange trill and says, "Oh no the Hai do lie and bluff, we're actually very good at bluffing, evolutionarily advantageous in the wild and all that. It's just seen as very immature and base. An adult should be able to conduct their lives without resorting to such primitive instincts. We learned that the hard way long ago, back before we holed up in our current borders."`

Reviewer 4: ```suggestion
			`	The barkeep chortles with a strange trill and says, "Oh no, the Hai do lie and bluff. We're actually very good at bluffing, evolutionarily advantageous in the wild and all that. It's just seen as very immature and base. An adult should be able to conduct their lives without resorting to such primitive instincts. We learned that the hard way long ago, back before we holed up in our current borders."`
```

			`	He looks at you intently before continuing, "You won't hear the tenets brought up very much, but if you look closely you'll see that the first three especially are ingrained in our society very deeply."`

> 			`	With a shrug he flips open his order pad and pats his belly. "Of course, my gran always said that any of them after the tenth were more guidelines than actual rules, and I certainly have my share of disagreements with thirteen. Now, before I talk your ears off, what'll I get for you?"`

Reviewer 4: `more guidelines than actual rules`
Is this a Pirates of the Caribbean reference? I hope it is

Author: It is. ;)

			label end
			`	He takes your order diligently and with a smile.`
				decline

############################################
#            Translator Required           #
############################################

#This uses the flag from receiving the Hai translator.
mission "ZZCU Mum Groups"
	minor
	source
		government "Hai"
		not attributes "station"
	to offer
		random < 1
		has "language: Wanderer"
	on offer
		conversation

> 			`A mall stall at the spaceport is trying to sell learning merchandise that's been condemned by the local conglomerate of mum groups. There's an argument and some flipped tables. (TBD)`

Reviewer 4: ```suggestion
			`A mall stall at the spaceport is trying to sell learning merchandise that's been condemned by the local conglomerate of mom groups. There's an argument and some flipped tables. (TBD)`
```
Still yet to be written? Also - American spelling prefers `mom` over `mum`.

Author: Yeah this is gonna get taken out in the clean-up and revisited
